internal/api/server.go

```diff
@@ -151,6 +151,9 @@ func (s *Server) CreateIdentity(ctx context.Context, request CreateIdentityReque
 
 // CreateClaim is claim creation controller
 func (s *Server) CreateClaim(ctx context.Context, request CreateClaimRequestObject) (CreateClaimResponseObject, error) {
+	const BJJSignatureProof2021 = "BJJSignatureProof2021"
```

Comment:
Can you use constants from here: https://github.com/iden3/go-schema-processor/blob/684a4b617a0eaf8452d4f3a65a4a6704ad53cbdf/verifiable/constants.go#L40

Reply:
Or the ones you defined in internal/api/api.gen.go